src/index.js

```diff
@@ -0,0 +1,234 @@
+import './style.css';
+import './assets/Spices & Flavors.png';
+import fetchLikes from './module/getLike.js';
+import getData from './module/getBaseData.js';
+import sendLike from './module/sendLike.js';
+import { involvementCommentUrl } from './module/apiData.js';
+import { postComment, printComment, fetchComments } from './module/comments.js';
+import { mealcounter, showCount } from './module/mealCounter.js';
+
+let arrayOfMeals = [];
+let arrayOfComments = [];
+let selectedId = '';
+
+const mainContainer = document.querySelector('.main-container');
+const infoPopup = document.querySelector('.info-popup-container');
+const backdrop = document.querySelector('.backdrop');
+const closeBtn = document.querySelector('.close-btn');
+const commentForm = document.querySelector('.comment-form');
+const formSubmitButton = document.querySelector('.comment-submit-button');
+const commentsList = document.querySelector('.comments-list');
+const commentLength = document.querySelector('.comment-length');
+
+// const getData = async () => {
+//   const result = await fetch(baseUrl);
+//   const { meals } = await result.json();
+//   return meals;
+// };
+
+// getData();
+
+const printFetchedComments = async () => {
+  arrayOfComments = [];
+  commentsList.innerHTML = '';
+  commentLength.textContent = 0;
+
+  try {
+    const request = await fetchComments(involvementCommentUrl, selectedId);
+    arrayOfComments = [...request];
+    console.log(arrayOfComments);
```

Comment:
Kindly remove this `console.log`, considering it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.